src/engine/blocks.js

```diff
@@ -463,18 +466,34 @@ class Blocks {
                 this.emitProjectChanged();
             }
             break;
+        case 'block_comment_collapse':
         case 'comment_collapse':
             if (this.runtime.getEditingTarget()) {
                 const currTarget = this.runtime.getEditingTarget();
                 if (currTarget && !Object.prototype.hasOwnProperty.call(currTarget.comments, e.commentId)) {
-                    log.warn(`Cannot change comment with id ${e.commentId} because it does not exist.`);
+                    log.warn(`Cannot collapse comment with id ${e.commentId} because it does not exist.`);
                     return;
                 }
                 const comment = currTarget.comments[e.commentId];
                 comment.minimized = e.newCollapsed;
                 this.emitProjectChanged();
             }
             break;
+        case 'block_comment_resize':
+        case 'comment_resize':
+            if (this.runtime.getEditingTarget()) {
+                const currTarget = this.runtime.getEditingTarget();
+                if (currTarget && !Object.prototype.hasOwnProperty.call(currTarget.comments, e.commentId)) {
```

Comment:
Do you know why we're calling `hasOwnProperty.call` instead of just calling `hasOwnProperty`? Are we expecting that the `hasOwnProperty` method on the object might be modified?

Reply:
No idea, I certainly hope not but this is just copypasta from the similar implementation above.